packages/e2e-test-app-fabric/test/__snapshots__/ViewComponentTest.test.ts.snap

```diff
@@ -0,0 +1,11349 @@
+                ],
+                "Visual Type": "SpriteVisual",
+              },
+              {
+                "Brush": {
+                  "Brush Type": "ColorBrush",
+                  "Color": "rgba(0, 0, 0, 255)",
+                },
+                "Offset": [
+                  -26,
+                  -26,
+                  0,
+                ],
+                "Opacity": 1,
+                "Size": [
+                  26,
+                  26,
+                ],
+                "Visual Type": "SpriteVisual",
+              },
+              {
+                "Brush": {
+                  "Brush Type": "ColorBrush",
+                  "Color": "rgba(0, 0, 0, 255)",
+                },
+                "Offset": [
+                  0,
+                  -6,
+                  0,
+                ],
+                "Opacity": 1,
+                "Size": [
+                  24,
+                  6,
+                ],
+                "Visual Type": "SpriteVisual",
+              },
+              {
+                "Brush": {
+                  "Brush Type": "ColorBrush",
+                  "Color": "rgba(0, 0, 0, 255)",
+                },
+                "Offset": [
+                  0,
+                  -6,
+                  0,
+                ],
+                "Opacity": 1,
+                "Size": [
+                  0,
+                  6,
+                ],
+                "Visual Type": "SpriteVisual",
+              },
+              {
+                "Brush": {
+                  "Brush Type": "ColorBrush",
+                  "Color": "rgba(0, 0, 0, 255)",
+                },
+                "Offset": [
+                  0,
+                  0,
+                  0,
+                ],
+                "Opacity": 1,
+                "Size": [
+                  0,
+                  44,
+                ],
+                "Visual Type": "SpriteVisual",
+              },
+            ],
+            "Offset": [
+              0,
+              0,
+              0,
+            ],
+            "Opacity": 1,
+            "Size": [
+              50,
+              50,
+            ],
+            "Visual Type": "SpriteVisual",
+          },
+          {
+            "Offset": [
+              0,
+              0,
+              0,
+            ],
+            "Opacity": 1,
+            "Size": [
+              0,
+              0,
+            ],
+            "Visual Type": "SpriteVisual",
+          },
+        ],
+        "Offset": [
+          390,
+          0,
+          0,
+        ],
+        "Opacity": 1,
+        "Size": [
+          50,
+          50,
+        ],
+        "Visual Type": "SpriteVisual",
+      },
+    ],
+    "Comment": "view-test-rounded-borders",
+    "Offset": [
+      0,
+      0,
+      0,
+    ],
+    "Opacity": 1,
+    "Size": [
+      916,
+      50,
+    ],
+    "Visual Type": "SpriteVisual",
+  },
+}
+`;
+
+exports[`View Tests Views can have shadows 1`] = `
```

Comment:
We should probably update the VisualTree snapshot code to collect shadow information, so that we actually ensuring the shadow props are set.